vault/provider_test.go

```diff
@@ -116,8 +116,7 @@ func getTestRMQCreds(t *testing.T) (string, string, string) {
 }
 
 // A basic token helper script.
-const tokenHelperScript = `
-#!/usr/bin/env bash
+const tokenHelperScript = `#!/usr/bin/env bash
```

Comment:
The extra newline added after the backtick caused the script to come out as 

```

#!/usr/bin/env bash
echo "helper-token"
```

With the shebang on the 2nd, rather than 1st, line. When executed as an exec by the cmd package for a user using fish shell the follow error silently happened:

```
> /usr/local/bin/fish -c $tmpScriptPath
Failed to execute process '$tmpScriptPath'. Reason:
exec: Invalid argument
The file '$tmpScriptPath' is marked as an executable but could not be run by the operating system.
```

Removing the line break here fixes that otherwise perplexing test failure that non-bash users may see